packages/contract-utils/access-control/src/access_control.rs

```diff
@@ -131,25 +133,31 @@ pub trait AccessControl {
     /// # Arguments
     ///
     /// * `e` - Access to Soroban environment.
-    /// * `caller` - The address of the caller, must be the admin.
     /// * `new_admin` - The account to transfer the admin privileges to.
     /// * `live_until_ledger` - The ledger number at which the pending transfer
     ///   expires. If `live_until_ledger` is `0`, the pending transfer is
     ///   cancelled. `live_until_ledger` argument is implicitly bounded by the
     ///   maximum allowed TTL extension for a temporary storage entry and
     ///   specifying a higher value will cause the code to panic.
     ///
-    /// # Errors
-    ///
-    /// * `AccessControlError::Unauthorized` - If the `caller` is not the admin.
-    /// * `AccessControlError::NoPendingAdminTransfer` - If tried to cancel the
-    ///   pending admin transfer when there is no pending admin transfer.
+    /// * [`stellar_role_transfer::RoleTransferError::NoPendingTransfer`] - If
```

Comment:
missing `# Errors`

Not a fan of listing those errors from another crate in this trait, although I don't have a better idea how to handle this, given the current structure.

Reply:
I have the EXACT same feelings :D